### DISCUSSION TITLE
when I change admin URL, my project API URL not found. how can I solve it?

### DISCUSSION THREAD

**Discussion by dxh-kaf** [cookiecutter/cookiecutter-django#4016] [Q&A]
Hi 

` path("admin/", admin.site.urls),` to  `path("", admin.site.urls),`  That much is fine.

but API Not Found  

or i need  `path("api/", include("config.api_router")),`

Note: when I use it  ` path("admin/", admin.site.urls),`  API is worked

**Comment by dacx** [comment]
Can you show us all your paths please?

**Comment by browniebroke** [comment]
> Can you show us all your paths please?

+1 it depends in which order the paths are defined. 

That might be because the admin has [a catch-all URL](https://docs.djangoproject.com/en/4.0/ref/contrib/admin/#django.contrib.admin.AdminSite.final_catch_all_view). It's it's located before the API one, then the catchall URL would match before the API. Note that this [view always matches](https://docs.djangoproject.com/en/3.2/releases/3.2/#django-contrib-admin), whether or not the URL exists on the admin:

> The admin now installs a final catch-all view that redirects unauthenticated users to the login page, regardless of whether the URL is otherwise valid. This protects against a potential model enumeration privacy issue.

Generally speaking, I would strongly advise against defining your admin path without prefix: `path("", admin.site.urls)`
